### DISCUSSION TITLE
ttk.Menubutton and menus???

### DISCUSSION THREAD

**Discussion by odds-get-evened** [alejandroautalan/pygubu-designer#275] [Q&A]
I saw a menubutton.ui file in the examples, and the XML structure makes sense, I just don't understand how you got there in the designer? Simply, how do we add a menu or even menu items to the Menubutton? I added the menu button, can't find anything in the properties section that allows us to use a menu id from a component in the project tree. the properties section would love that.

![image](https://github.com/user-attachments/assets/61d42d37-c4bb-4888-a871-58c712688113)

**Comment by alejandroautalan** [comment]
Hello @odds-get-evened, thanks for trying pygubu.

You found a bug! Sorry for the inconvenience.
The designer is supposed to allow adding a Menu object to the Menubutton. I'll try to fix this as soon as possible.
In the meantime you can associate a menu by code. Example [here](https://github.com/alejandroautalan/pygubu-designer/tree/35685af8fc916b213eb897453ff8218e8a7d28c1/examples/menubutton).

I created a new issue here #276.

Regards!
Alejandro A.

**Reply by alejandroautalan** [reply]
Fixed in pygubu 0.35.6
